Foundation/CGFloat.swift

```diff
@@ -611,6 +611,7 @@ public func %=(lhs: inout CGFloat, rhs: CGFloat) {
     fatalError("%= is not available.")
 }
 
+#if !os(Android)
```

Comment:
I’ll have another look at it. There are some tricky type conversions to get right.
 Add a line note